apps/functions/applications-migration/common/constants.js

```diff
@@ -0,0 +1,2 @@
+export const ODW_GENERAL_CASE_REF = 'General';
+export const BO_GENERAL_CASE_REF = 'GS5110001';
```

Comment:
Can we put these in the common `packages/applications` package please? (we should then use these in API + WEB, I think Francesco was going to make a ticket for that) - else this would be the third copy of this string!

Reply:
I've created the addition to the `packages/applications`, but I haven't gone through the entire codebase replacing references to it yet. I started doing it and the # of file changes got a bit much, so will create a separate pr for it

Reply:
Yeah that's fine, let's do that separately.